content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,76 @@
+# Internal link checking tool
+
+You can use [htmltest](https://github.com/wjdp/htmltest) to check for broken links in [`/content/en/`](https://git.k8s.io/website/content/en/). This is useful when refactoring sections of content, moving pages around, or renaming files or page headers.
+
+## How the tool works
+
+`htmltest` scans links in the generated HTML files of the kubernetes website repository. It runs using a `make` command which does the following:
+
+- Builds the site and generates output HTML in the `/public` directory of your local `kubernetes/website` repository
+- Pulls the `wdjp/htmltest` Docker image
+- Mounts your local `kubernetes/website` repository to the Docker image
+- Scans the files generated in the `/public` directory and provides command line output when it encounters broken internal links
+
+## What it does and doesn't check
+
+The link checker scans generated HTML files, not raw Markdown. The htmltest tool depends on a configuration file, [`.htmltest.yml`](https://git.k8s.io/website/.htmltest.yml), to determine which content to examine.
+
+The link checker scans the following:
+
+- All content generated from Markdown in [`/content/en/docs`](https://git.k8s.io/website/content/en/docs/) directory, excluding:
+  - Generated API references, for example https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
+- All internal links, excluding:
+  - Empty hashes (`<a href="#">` or `[title](#)`) and empty hrefs (`<a href="">` or `[title]()`)
+  - Internal links to images and other media files
+
+The link checker does not scan the following:
+
+- Links included in the top and side nav bars, footer links, or links in a page's `<head>` section, such as links to CSS stylesheets, scripts, and meta information
+- Top level pages and their children, for example: `/training`, `/community`, `/case-studies/adidas`
+- Blog posts
+- API Reference documentation, , for example https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```

Comment:
nit:
```suggestion
- API Reference documentation; for example: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```

Reply:
Resolved locally ergo not committing this suggestion :)